src/app/components/List/EditionsList.jsx

```diff
@@ -1,35 +1,68 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import FeatureFlags from 'dgx-feature-flags';
+import featureFlagConfig from '../../../../featureFlagConfig';
+import { checkFeatureFlagActivated } from '../../util/Util';
 import * as DS from '@nypl/design-system-react-components';
 import EditionCard from '../Card/EditionCard';
+import FormattedAPACitation from '../Citations/APACitation';
+
+import config from '../../../../appConfig';
 
 
 class EditionsList extends React.Component {
   constructor(props) {
     super(props);
     this.props = props;
 
+    this.state = { isFeatureFlagsActivated: {} };
+
     this.getAllEditionsData = this.getEditionCardList.bind(this);
+    this.onFeatureFlagsChange = this.onFeatureFlagsChange.bind(this);
+  }
+
+  componentDidMount() {
+    FeatureFlags.store.listen(this.onFeatureFlagsChange.bind(this));
+
+    checkFeatureFlagActivated(
+      featureFlagConfig.featureFlagList, this.state.isFeatureFlagsActivated,
+    );
+  }
+
+  onFeatureFlagsChange() {
+    // eslint-disable-next-line react/no-unused-state
+    this.setState({ featureFlagsStore: FeatureFlags.store.getState() });
   }
 
   getEditionCardList(work, eReaderUrl, referrer) {
     return work.editions.map(
       (edition, index) => {
         const showRequestButton = this.props.getRequestEditionButton(edition);
         const editionData = EditionCard.getEditionData(work, edition, eReaderUrl, referrer, showRequestButton);
+
+        const shouldShowCitations = FeatureFlags.store._isFeatureActive(config.displayCitations.experimentName);
+        const citationElement = shouldShowCitations ? (
+          <FormattedAPACitation
```

Comment:
Here I'd say something like parsing out initials is part of "display", even though /technically/ it might not be.  

Also, do we expect last names and first names to always be well labeled?

Reply:
We definitely don't expect last names/first names to be well labeled, the script is doing its best to identify them, and seems to be handling it okay.

I think in general we want this to be reusable, but it's hard because each citation can't be counted on to share any formatting. It would be great to have an element that can be passed a `style` prop like `<FormattedCitation style='apa' title={} {..etc}>` but I can't quite see how to achieve that with React, any thoughts?

I do think that it should only be passed the formatted logic.

Reply:
That's with the caveat that <FormattedCitation> shouldn't just implement different methods for each style, but invoke the desired class and return that.

Reply:
OH in terms of the `APA`, `MLA`, etc components being reuseable with each other, I think we should just ... not do that.  They might share methods to get a well formatted title or something, but I think it's perfectly OK (maybe better?) to have a separate `APACitation` and `MLACitation`, etc, etc.

Reply:
The reuseability comment is more about how much this should be able to be ripped out of RN and still work, vs tied into the way RN works.